changelog.d/graphql_endpoint_toggle.enhancement.md

```diff
@@ -0,0 +1,3 @@
+Added boolean `graphql` field to the api configuration to toggle graphql endpoint on/off in Vector.
```

Comment:
nit: The "in Vector" part is redundant, as that is the overall context of all such changes.
```suggestion
Added a boolean `graphql` field to the api configuration to allow disabling the graphql endpoint.
```